gcc/rust/checks/errors/rust-feature.h

```diff
@@ -57,15 +58,15 @@ class Feature
 	   const Optional<CompileOptions::Edition> &edition,
 	   const char *description)
     : m_state (state), m_name (name), m_name_str (name_str),
-      m_rustc_since (rustc_since), issue (issue_number), edition (edition),
+      m_rustc_since (rustc_since), m_issue (issue_number), edition (edition),
       m_description (description)
   {}
 
   State m_state;
   Name m_name;
   std::string m_name_str;
   std::string m_rustc_since;
-  uint64_t issue;
+  uint64_t m_issue;
```

Comment:
We probably need to consider our naming conventions for fields like this @tschwinge or @dkm what is the gnu standard for class fields? I guess that we don't do 'm_'

This is not something to change for this PR we might raise an issue to fix it in a future PR.

Reply:
In fact there are very many instances of `m_[...]` member variables In GCC, so that's fine.